translations/messages.en.yaml

```diff
@@ -292,7 +292,7 @@ comment: Comment
 post: Post
 ban_expired: Ban expired
 purge: Purge
-send_message: Send message
+send_message: Send private message
```

Comment:
I don't believe we should use the word private, as other software doesn't for the reason that it implies the messages are encrypted or secure, instead the word used is "direct message"

Reply:
agree with @e-five256, there's nothing private about AP at all, lol.

Reply:
I see. Let's call them direct message instead.

Reply:
Fixed.